src/patternfly/components/Card/card.scss

```diff
@@ -380,6 +384,10 @@
   font-size: var(--pf-c-card__footer--FontSize);
 }
 
+input:focus + .pf-c-card {
```

Comment:
This selector is likely OK, though it is a bit generic and has the potential to cause an issue. Can we give it a class? 

Also toggling a `border` on a normal element like that will cause the content to shift a bit. Curious if you've tested it and seen if it shifts by the size of the border width? Typically we'll use a pseudo element for stuff like that, looks like we could use `::after` here.

Also I'm not completely familiar with this feature. Is this input is for screen readers only - not keyboard users?

Reply:
Correct screen readers only, keyboard users that aren't using a screen reader won't be able to reach the hidden input.

And good points, I'll give it a class and use `::after` as I do see it shift a bit when the border is applied now that you mention it.